mlsdbtool/main.cpp

```diff
@@ -1433,8 +1532,18 @@ void printRegionHelp()
                     " file `mlsdb.data` which contains a mapping from (uint32) cell id"
                     " to (double,double) latitude,longitude for all cells known"
                     " in the mls.csv within the bounding-box for that region.\n"
-                    "One special region is the `Devel` region which contains just"
-                    " Finland, Australia and India.\n");
+                    "A few special regions are added:\n"
```

Comment:
Bit of a nitpick, but only that devel is a 'special' region. The other ones are actual regions :)